modules/apps/portal-scheduler/portal-scheduler-quartz/src/test/java/com/liferay/portal/scheduler/quartz/internal/QuartzSchedulerEngineTest.java

```diff
@@ -478,6 +477,23 @@ private void _assertTriggerState(
 		Assert.assertEquals(expectedTriggerState, jobState.getTriggerState());
 	}
 
+	private void _setProps(Props props) {
```

Comment:
@JanisZhang Please make scheduler enabled false by default in `setUp()` so that you can always activate the `QuartzSchedulerEngine`.

In tests, when you need to modify the int values, just directly modify the fields.

Only in the tests that test activation/deactivation, you set a new props and reinvoke activate.